src/transformers/models/gpt2/tokenization_gpt2_fast.py

```diff
@@ -146,6 +146,17 @@ def __init__(
             **kwargs,
         )
 
+        if kwargs.pop("add_bos_token", False):
```

Comment:
Gives:

```
Traceback (most recent call last):
  File "/home/patrick/hf/lib/python3.9/site-packages/IPython/core/interactiveshell.py", line 3369, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-8-69932b6461cf>", line 1, in <cell line: 1>
    gpt2 = GPT2TokenizerFast.from_pretrained("facebook/opt-30b")
  File "/home/patrick/python_bin/transformers/tokenization_utils_base.py", line 1783, in from_pretrained
    return cls._from_pretrained(
  File "/home/patrick/python_bin/transformers/tokenization_utils_base.py", line 1928, in _from_pretrained
    tokenizer = cls(*init_inputs, **init_kwargs)
  File "/home/patrick/python_bin/transformers/models/gpt2/tokenization_gpt2_fast.py", line 151, in __init__
    raise ValueError(
ValueError: Currenty GPT2's fast tokenizer does NOT support adding a BOS token.Instead you should use GPT2's slow tokenizer class `GPT2Tokenizer` as follows: 
`GPT2Tokenizer.from_pretrained('facebook/opt-incorrect')`
or
`AutoTokenizer.from_pretrained('facebook/opt-incorrect', use_fast=False)`
This issue will be fixed soon, see: https://github.com/huggingface/tokenizers/pull/1005. so that the fast tokenizer works correctly.
```

Reply:
Interesting so the error wasn't raised correctly when loading the tokenizer configuration?